packages/react-instantsearch-nextjs/package.json

```diff
@@ -50,11 +50,11 @@
   },
   "devDependencies": {
     "instantsearch.js": "4.75.7",
-    "next": "13.5.1",
+    "next": "15.1.4",
     "react-instantsearch-core": "7.13.10"
   },
   "peerDependencies": {
-    "next": ">= 13.4 < 15",
+    "next": ">= 13.4 < 16",
```

Comment:
Does it still work with next 14 ? What happens when `use` receives a non-promise value ?

Reply:
If it doesn't work we could check whether `headers()` returns a promise or not

Reply:
Ah yes, you're right, `use` would error out when a non Promise is passed in. I'll add a check for it 👍